src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -224,6 +228,15 @@ private void createAndAddNavItems() {
         aboutNav = createNavItem(resourceBundle.getString("layoutAbout"),
             Icons.getIcon(INFO),
             aboutView);
+        upgradeNav = createNavItem(resourceBundle.getString("upgrade"),  Icons.getIcon(Icons.LINK),
+            ((event) -> executor.submit(() -> {
+            try {
+                Desktop.getDesktop().browse(new URI("https://qabel.de/de/qabelnow"));
```

Comment:
Technically this should be a localized string. German locale -> .de/de/, english -> .de/en/